### PR TITLE
Fix a few docstrings

### PR DESCRIPTION
**Pull request by kinow**
This is a small change with no associated Issue.

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).

- [x] Does not need tests (docs only).

- [x] No change log entry required (why? e.g. invisible to users).

- [x] No documentation update required.

### CODE REVIEW

**Comment by kinow** [comment]
Probably 1 review will do...

File: cylc/flow/network/schema.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def parse_node_id(item, node_type=None):
     """Parse definition, job, or proxy id argument returning components.
 
     Args:
-        item: An string representing a node ID. Jobs fill out
+        item (str): An string representing a node ID. Jobs fill out
```
Comment (dwsutherland) [inline cylc/flow/network/schema.py]: Two things:
- Self correction, should be: `A string representing. . .`
- Why are you getting rid of ` (str)`? it's in the same style as `network/server.py` end-point methods?

Maybe I don't understand docstrings properly!

File: cylc/flow/network/schema.py [hunk 2]
```diff
@@ -88,7 +88,7 @@ def parse_node_id(item, node_type=None):
...
             cycle%name%num first, cycle is irrelevant to Def
```
Comment (dwsutherland) [inline cylc/flow/network/schema.py]: I didn't update the delimiter `%` to `|` .. I suppose f-string doc (if possible?) would be inefficient (i.e. using `ID_DELIM`)

File: cylc/flow/network/schema.py (hunk 1 above)
Comment (kinow) [reply cylc/flow/network/schema.py]: >Self correction, should be: A string representing. . .

Ops! Correcting now.

>Why are you getting rid of (str)? it's in the same style as network/server.py end-point methods?

Hmm, I thought I was adding it, not getting rid

File: cylc/flow/network/schema.py (hunk 2 above)
Comment (kinow) [reply cylc/flow/network/schema.py]: Will fix the "An string" here too. Not sure about the delimiter and f-strings though, I'm a bit out of the loop (wasn't really using the code, just updated the docs)

File: cylc/flow/network/schema.py (hunk 1 above)
Comment (dwsutherland) [reply cylc/flow/network/schema.py]: > > Self correction, should be: A string representing. . .
> 
> Ops! Correcting now.
> 
> > Why are you getting rid of (str)? it's in the same style as network/server.py end-point methods?
> 
> Hmm, I thought I was adding it, not getting rid

Double Oppsie! Yes please add!

File: cylc/flow/network/schema.py (hunk 2 above)
Comment (dwsutherland) [reply cylc/flow/network/schema.py]: > Will fix the "An string" here too. Not sure about the delimiter and f-strings though, I'm a bit out of the loop (wasn't really using the code, just updated the docs)

Was just mentioning it while you're fixing my mistakes XD

Comment (kinow) [reply cylc/flow/network/schema.py]: Oh, same error as above haha, thought this comment was on another line. Pushing updated commit.

**Comment by kinow** [comment]
Updated!

**Comment by dwsutherland** [comment]
Also updated!  (not too cheeky I hope 😛 )

**Comment by matthewrmshin** [comment]
I'm going to merge this to reduce the number of PRs.

**Pull request merged**